rtv/subreddit.py

```diff
@@ -168,6 +187,16 @@ def open_subscriptions(self):
             self.refresh_content(order='ignore',
                                  name=page.subreddit_data['name'])
 
+    def on_frontpage(self):
+        "Returns whether the current page is the front page"
+        return self.content.name == '/r/front'
+
+    def set_last_subreddit(self):
```

Comment:
You don't need to use a getter/setter here, just access the variable directly from your show_frontpage() function.